app/views/users/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for @user do |f| %>
+  <%= f.label :username %>
+  <%= f.text_field :username %>
+  <%= f.label :password %>
+  <%= f.password_field :password %>
+  <%= f.label :address %>
+  <%= f.text_field :address %>
```

Comment:
Pulling the address at checkout is a sweet idea.  We could leave this as an optional field when they sign up and if they do we can give them an option to auto populate it on checkout with a simple drop down possibly too.